groovy/gradle.java/nbproject/project.properties

```diff
@@ -20,7 +20,10 @@ javac.source=1.8
 javac.compilerargs=-Xlint -Xlint:-serial
 nbm.module.author=Laszlo Kishalmi
 
+test-unit-sys-prop.test.netbeans.dest.dir=${netbeans.dest.dir}
+test-unit-sys-prop.java.awt.headless=true
```

Comment:
Any reason to force this to headless? If not required, I would try to stay with defaults.

Reply:
@matthiasblaesing - if you can run a test headless, you should.  In particular, I know there are tests in exception handling code and the dialogs api for whether the toolkit is headless.  While there are tests that must open a UI a drive it, this tends to hijack focus on developer machines unless you run tests in a virtual X framebuffer.  Since thrown exceptions can result in dialogs unexpectedly popping up, any test that *can* run headless should prefer that, and only tests that simply must interact with windows and will fail otherwise should not be run headless.

Reply:
What made you think I disagree with the assessment, that unittests should be run without GUI interaction? I just don't like setting properties all through the codebase, because they have the tendency not to be maintained. We learned  in the discussion about the `javac.source`, that people forget why a property was set. We also learned, that system default can be better, then your own implementation, as seen in the discussion about the default heap use calculation, which was switch to use the JDK default.

Reply:
The headless true was just copied from another module which dealt with project ui module. As I need to actually open the project to get the information from Gradle instead of the heuristics.